cluster/gce/util.sh

```diff
@@ -754,6 +754,19 @@ function construct-linux-kubelet-flags {
   # Keep in sync with the mkdir command in configure-helper.sh (until the TODO is resolved)
   flags+=" --cert-dir=/var/lib/kubelet/pki/"
 
+  # Enabling KubeletCredentialProviders and DisableKubeletCloudCredentialProviders
+  # enables kubelet to use out-of-tree auth credential provider instead of in-tree 
+  # auth credential provider.
+  # https://kubernetes.io/docs/tasks/kubelet-credential-provider/kubelet-credential-provider
+  if [[ "${ENABLE_AUTH_PROVIDER_GCP:-false}" == "true" ]]; then
+    flags+="  --feature-gates=DisableKubeletCloudCredentialProviders=true,KubeletCredentialProviders=true"
```

Comment:
`KubeletCredentialProviders` is already defaults to true [1], it can probably be omitted? 

[1] https://github.com/kubernetes/kubernetes/blob/5d446b205e308e54119320b5645fe17ce88d39cf/pkg/features/kube_features.go#L937

Reply:
yeah i guess it is fine to remove  `KubeletCredentialProviders`, coz if in any case `KubeletCredentialProviders` is set to false, setting `DisableKubeletCloudCredentialProviders=true` will fail all image pulls and probably the test anyway (since no credential provider will be available)